src/main/scala/viper/gobra/frontend/info/implementation/typing/IdTyping.scala

```diff
@@ -174,6 +175,39 @@ trait IdTyping extends BaseTyping { this: TypeInfoImpl =>
 
     case Import(decl, _) => ImportT(decl)
 
+    case Wildcard(decl, _) => getWildcardType(decl)
+
     case _ => violation("untypable")
   }
+
+  /**
+    * Gets the type of a blank identifier if it occurs in the `left` list
+    */
+  def getBlankAssigneeType(n: PNode, left: Vector[PNode], right: Vector[PExpression]): Type = {
```

Comment:
How is this different from `getBlankIdType` from ExprTyping.scala?

Reply:
`getBlankIdType` accepts a `PBlankIdentifier b` and does a case split on the expressions where `b` could occur. It then delegates all the logic of finding the value of the blank identifier to the `getBlankAssigneeType` method. `getWildcardType` in file `IdTyping.scala` has an analogous behavior for `PWildcard`s